cmd/dptp-controller-manager/main.go

```diff
@@ -311,7 +311,9 @@ func main() {
 		if cluster == opts.registryClusterName {
 			registryMgr = mgr
 		} else {
-			allClustersExceptRegistryCluster[cluster] = mgr
+			if cluster != "api.ci" {
+				allClustersExceptRegistryCluster[cluster] = mgr
```

Comment:
No, do it inside the controller specifically, this map might be used elsewhere

Reply:
Checked. only this controller.
hesitating as well.
Modifying.